assets/javascripts/running.js

```diff
@@ -263,6 +264,7 @@ function initLivestream() {
     }
     setResolution(livestream_size);
     sel_resolution.value = livestream_size;
+    */
```

Comment:
It means asdil12 can disable code and you split the commit into "stuff to push in a PR" and "stuff to keep for later in my local WIP commits".